server/local/strings.yml

```diff
@@ -16,5 +16,6 @@ strings: {
   canvasUrl: "https://umich.instructure.com/",
   ctoolsUrl: "https://ctools.umich.edu/portal",
   noCourses: "You seem to have no courses this term",
-  termFailure: "We are not able to access your term enrollment information at the moment. Please try again later."
+  termFailure: "We are not able to access your term enrollment information at the moment. Please try again later.",
+  dashIntroMessage: "Lorem ipsum dolor <strong>sit amet</strong>, quam phasellus viverra, et malesuada, vivamus elit pede ligula amet cubilia, in eu. Pulvinar praesent nec vivamus arcu qui sodales. Commodo est tempus, sodales nisl lorem luctus maecenas. Aptent sapien lectus tincidunt volutpat pede luctus, eros libero, lacinia faucibus luctus integer netus. Sapien elit magna convallis, fringilla vestibulum, tincidunt morbi tortor. ",
```

Comment:
Looks fine for now.  We probably don't want to have to do builds to change this text in the long run.